frontend/test/components/form/FormTextEntity.spec.ts

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+import FormTextEntity from "@/components/form/FormTextEntity.vue";
+import render from "@/test/render";
+import { fireEvent, screen } from "@testing-library/vue";
+import { describe, expect, it, vi } from "vitest";
+
+import { getEnglishText } from "~/utils/i18n";
+
+describe("FormTextEntity component", () => {
+  const mockHandleSubmit = vi.fn();
+
+  const defaultProps = {
+    handleSubmit: mockHandleSubmit,
+    submitLabel: "i18n.components.modal._global.update_texts",
+    getInvolvedLabel: "i18n.components._global.get_involved",
+    getInvolvedUrlLabel:
+      "i18n.components.modal_text_organization.join_organization_link",
+  };
+
+  it("shows validation error when neither getInvolved text nor URL is provided", async () => {
+    await render(FormTextEntity, {
+      props: defaultProps,
+    });
+
+    const descriptionInput = screen.getByLabelText(
+      getEnglishText("i18n._global.description")
+    );
+    await fireEvent.update(descriptionInput, "Test description");
+
+    const submitBtn = screen.getByRole("button", {
+      name: "Submit the form",
+    });
+
+    await fireEvent.click(submitBtn);
+
+    const getInvolvedError = await screen.findByTestId(
+      "form-item-getInvolved-error"
+    );
+    const getInvolvedUrlError = await screen.findByTestId(
+      "form-item-getInvolvedUrl-error"
+    );
+
+    expect(getInvolvedError.textContent).toBe(
+      getEnglishText(
+        "i18n.components.form_text_entity.get_involved_text_or_url_required"
+      )
+    );
+    expect(getInvolvedUrlError.textContent).toBe(
+      getEnglishText(
+        "i18n.components.form_text_entity.get_involved_text_or_url_required"
+      )
+    );
+  });
+
+  it("allows submission when getInvolved text is provided", async () => {
+    await render(FormTextEntity, {
+      props: defaultProps,
+    });
+
+    // fill in thr required fields
```

Comment:
Not a big deal for this PR, @OmkarSathish, but for future ones we normally do the following for comments regardless of if it's backend or frontend:
- If it's its own line, then capitalize, end with a period and make it a full sentence
- If it's an inline comment, then no capitalization, no period and keep it short

We can handle it though if you ever forget :)